src/mindlessgen/prog/config.py

```diff
@@ -560,6 +561,22 @@ def xtb_path(self, xtb_path: str | Path):
             raise TypeError("xtb_path should be a string.")
         self._xtb_path = xtb_path
 
+    @property
+    def level(self):
+        """
+        Get the refinement level.
+        """
+        return self._level
+
+    @level.setter
+    def level(self, level: int):
+        """
+        Set the refinement level.
+        """
+        if not isinstance(level, int):
+            raise TypeError("Refinement level should be a string.")
```

Comment:
```suggestion
            raise TypeError("xtb level level should be an integer.")
```

Probably copied and not adjusted ;)

Reply:
is the text now "xtb level level should be an integer? ", at least this stands in the changed line

Reply:
Don't understand the question – if you commit the suggestion, then yes; currently not. 😄

Reply:
Good catch, should be only a single "level" obviously.